content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,262 @@
+        </td>
+        <td>Beta</td>
+        <td>v0.2.2</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+          (<a href="https://github.com/kubeflow/metadata">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mpi/">MPI training: MPI 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mpi-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mxnet/">MXNet training: MXNet 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mxnet-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook web
+          app</a>
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/jupyter-web-app">GitHub</a>)
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook 
+          controller</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/notebook-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/overview/pipelines-overview/">Pipelines</a>
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/multi-user-overview/">Profile 
+          Controller for multi-user isolation</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/pytorch/">PyTorch training: PyTorch operator</a> 
+          (<a href="https://github.com/kubeflow/pytorch-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfbatchpredict/">TensorFlow 
+          batch prediction</a>
+          (<a href="https://github.com/kubeflow/batch-predict">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfserving_new/">TensorFlow
+          serving</a></td>
+        <td>Stable</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/tftraining/">TensorFlow training:
+          TFJob operator</a>
+          (<a href="https://github.com/kubeflow/tf-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
```

Comment:
/cc @richardsliu please confirm this.

Reply:
Yes: https://github.com/kubeflow/manifests/blob/e9dfeb30abcd12a920ad8f8510f26a167390e5fc/tf-training/tf-job-crds/overlays/application/application.yaml#L13